R/create_default_parameters.R

```diff
@@ -348,54 +348,103 @@ create_default_fleet <- function(fleets,
     dplyr::pull(type) |>
     unique()
 
+  #Get data likelihood distributions assigned for this fleet
+  distribution_names_for_fleet <- names(fleets[[fleet_name]][["data_distribution"]])
+
   # Determine default fleet parameters based on types of data present
-  # FIXME: allow for a fleet to have both landings and index data
-  process_default <- if ("landings" %in% data_types_present) {
-    list(
-      log_Fmort.value = log(rep(0.00001, get_n_years(data))),
-      log_Fmort.estimated = TRUE
-    )
-  } else {
-    list(
+  if ("index" %in% data_types_present &&
```

Comment:
seems undesirable to use "index" for one category and "Index" for another.

Reply:
Good point Tim, I was just following the existing format for this. Maybe we can brainstorm better naming on Thursday.